week-4-practise/level-2/exercise-1.js

```diff
@@ -1,22 +1,28 @@
 /*
-Create an object that acts like a water bottle. 
-It will need a volume key to store how full or empty the bottle is. 
-It will be 100 when full and 0 when empty. 
-Give your water bottle methods for filling it up, 
+Create an object that acts like a water bottle.
+It will need a volume key to store how full or empty the bottle is.
+It will be 100 when full and 0 when empty.
+Give your water bottle methods for filling it up,
 drinking some of it, and emptying it.
 
-We made a start on this for you here: 
+We made a start on this for you here:
 */
 
 var bottle = {
   volume: 0,
   fill: function() {
+    return (this.volume = 100);
+
     // calling this function should make you bottles volume = 100;
   },
   drink: function() {
+    return (this.volume -= 10);
     // calling this function should decrease your bottles volume by 10;
   },
   empty: function() {
+    if (bottle.volume === 0) {

```

Comment:
You do need a return here. :)

Reply:
i think just need just this.volume=...

Reply:
Yes, `return this.volume === 0;` is a good short way to write it.